src/components/ConsentModal.vue

```diff
@@ -53,12 +53,11 @@ onMounted(() => {
   dialogVisible.value = true;
 
   const acceptIcon = computed(() => (isSubmitting.value ? 'pi pi-spin pi-spinner mr-2' : 'pi pi-check mr-2'));
+  const header = props.consentType.includes('consent') ? `Consent` : `Assent`;
```

Comment:
It seems like you got rid of the translations in this refactor. Was that intentional? If so, can you justify the removal?